CHANGELOG.md

```diff
@@ -4,7 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
 ## Unreleased
-### Added
+### Added/removed
+- Implementation of the Marshal/Unmarshal library
+- Implementation of the Enhanced System API (ESAPI)
+- New implemetation of the TPM Command Transmission Interface (TCTI) with two
+  TCIT modules tcti-device and tcti-mssim
+- New directory layout
+- Updated documentation with new doxygen and updated man pages
+- Support for Windows build with Visual Studio and clang
+- Implementation of the new Attached Component (AC) commands
+- Implementation of the new TPM2_CCPolicyAuthorizeNV command
+- Implementation of the new TPM2_CC_CreateLoaded command
+- Implementation of the new TPM2_CC_PolicyTemplate command
```

Comment:
The commands are just called TPM2_PolicyTemplate, etc...

Reply:
done